app/package.json

```diff
@@ -113,6 +113,7 @@
     "markdown-it": "^14.1.0",
     "moment": "2.29.4",
     "mustache": "4.2.0",
+    "nano": "^10.1.4",
```

Comment:
Why the added dep on nano? Doesn't seem to be used (and we have pouchdb for that)

Reply:
Accident